sites/fast-website/src/app/components/feature-card/feature-card.ts

```diff
@@ -0,0 +1 @@
+export class FeatureCard {}
```

Comment:
What's the scenario here? Is this being used at all?

Reply:
It is, but this wasn't actually set up right.  I've updated it.